blackjack.rb

```diff
@@ -93,6 +105,13 @@ def player_hit_or_stay(current_hand)
     end
   end
 
+#cpu doesn't handle aces (unless I missed it).
+# I did something like this, but it assumes ace will auto change so you don't bust
+# if hand.any?{ |c| c.face == 'A' }
```

Comment:
thats just what mind does within a method in my dealer instance. so yeah, in your case it would be something like dealah.hands_array[0].cars.any?. I added something like it at first, but then I realized that it assumes aces will adjust if you would bust with 11.